pkgs/development/python-modules/basic-colormath/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  setuptools-scm,
+  pillow,
+  nix-update-script,
+}:
+buildPythonPackage rec {
+  pname = "basic-colormath";
+  version = "0.5.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "basic_colormath";
+    hash = "sha256-p/uNuNg5kqKIkeMmX5sWY8umGAg0E4/otgQxhzIuo0E=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [ pillow ];
+
```

Comment:
Are we executing the tests?

https://github.com/ShayHill/basic_colormath/tree/main/tests